dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -845,4 +845,14 @@ private Constants() {
     public static final String SECURITY_CONFIG_TYPE_PASSWORD = "PASSWORD";
 
     public static final String SECURITY_CONFIG_TYPE_LDAP = "LDAP";
+    /**
+     * Task Types
+     */
+    public static final String TYPE_UNIVERSAL = "universal";
+    public static final String TYPE_DI = "di";
+    public static final String TYPE_CLOUD = "cloud";
+    public static final String TYPE_LOGIC = "logic";
+    public static final String TYPE_DQ = "dq";
```

Comment:
please use the full name or add comment.

Reply:
done